python/ray/autoscaler/aws/example-auto-instance-type.yaml

```diff
@@ -0,0 +1,48 @@
+# EXPERIMENTAL: an example of configuring a mixed-worker cluster. Currently
+# multi-worker autoscaling only works if you use the request_resources() call.
+cluster_name: auto_instance_type
+min_workers: 1
+max_workers: 40
+
+# Cloud-provider specific configuration.
+provider:
+    type: aws
+    region: us-west-2
+    availability_zone: us-west-2a
+
+# Tell the autoscaler the allowed node types and the resources they provide.
+# Setting this configuration enables the Tune resource demand scheduler.
+available_instance_types:
+    m4.xlarge:
+        resources: {"CPU": 4}
```

Comment:
Is it necessary to specify the resource specs for each instance type? Couldn't we look this information up? This seems like it could be brittle if the resource spec is inaccurate.

Reply:
We can, but it would require additional API calls specific to each cloud or storing the metadata somewhere. It's probably easiest to specify it manually for now.